examples/example_config_files/public_TrueCoaxial.yaml

```
@@ -26,23 +26,25 @@ medium: vacuum
 detectors:
 - semiconductor:
     material: HPGe
-    temperature: 77
+    temperature: 90
     impurity_density:
-      name: constant
-      value: 1e8cm^-3
+      name: PtypePNjunction
+      Li_annealing_temperature: 623
+      Li_annealing_time: 1080
+      bulk_imp: -1e10cm^-3
```

As said, maybe this can become any other impurity density, something like
```suggestion
      bulk_imp:
        name: constant
        value: -1e10cm^-3
```
and then this is actually parsed as another (already existing) impurity density (?)